bundles/org.openhab.binding.speedtest/src/main/resources/OH-INF/i18n/speedtest_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
```

Comment:
You can delete this file

Reply:
done and replaced via `mvn i18n:generate-default-translations`